packages/rancher-logging/rancher-logging.patch

```diff
@@ -100,6 +100,10 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
  
  affinity: {}
  
+@@ -55,2 +60,2 @@
```

Comment:
Do we have a policy for the charts repo on if PSP should be enabled by default? We may also want to consider making this option available on the chart install page.

Reply:
In rancher-monitoring and longhorn we have PSPs enabled as default.